tools/issue-labeler/src/AzureRAGService/AzureRAGService.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
If we're using `Directory.Build.props` for common properties, we should integrate these with it.